<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login - WEHA Smart POS</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="MainFrame">
     <property name="styleSheet">
      <string notr="true">QFrame#MainFrame{
	background-color: rgb(45, 65, 90);
	border-radius: 5px;
	border-color: rgb(107, 108, 108);
	border-width: 1px;
	
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
	background-color: rgb(78, 93, 108);
	border-radius: 5px;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QFrame" name="frame">
           <property name="styleSheet">
            <string notr="true">QFrame{
	background-color: rgb(56, 67, 81);
	border-radius: 5px;
}
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QPushButton" name="ConfigurationPushButton">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
    color: rgb(255, 255, 255);
	background-color: rgb(229, 88, 23);
    border-radius: 5px;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:hover{
    color: rgb(255, 255, 255);
	background-color: rgb(179, 70, 19);
    border-radius: 5px;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #31363b;
	background-color:#ffff00;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(225, 228, 228);
    background-color:  rgb(255, 255, 255);
    border-radius: 3;
    border: 2px solid rgb(225, 228, 228);
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../WehaSmartPos/icons/settings.png</normaloff>../WehaSmartPos/icons/settings.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="ExitButton">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Futura</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
    color: rgb(255, 255, 255);
	background-color: rgb(229, 88, 23);
    border-radius: 5px;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:hover{
    color: rgb(255, 255, 255);
	background-color: rgb(179, 70, 19);
    border-radius: 5px;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #31363b;
	background-color:#ffff00;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(225, 228, 228);
    background-color:  rgb(255, 255, 255);
    border-radius: 3;
    border: 2px solid rgb(225, 228, 228);
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../WehaSmartPos/icons/logout.png</normaloff>../WehaSmartPos/icons/logout.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="styleSheet">
            <string notr="true">QFrame{
	background-color: rgb(56, 67, 81);
	border-radius: 5px;
}
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Futura</family>
                <pointsize>32</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
	color: rgb(253, 133, 51);
}</string>
              </property>
              <property name="text">
               <string>Smart PARKING</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame{
	background-color: rgb(56, 67, 81);
	border-radius: 5px;
}
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>10</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLineEdit" name="EmailEdit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>-1</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLineEdit{  
	background-color: rgb(204, 204, 204);
     border: 2px solid gray;
	border-radius: 10px;
	padding: 0 8px;
	selection-background-color: darkgray;
	font-size: 16px;
    color: rgb(255,255,255);
}
QLineEdit:focus {
     background-color: rgb(213, 83, 21);
     color: rgb(255,255,255);
}
</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="placeholderText">
                 <string>LOGIN</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="EmailNumpadButton">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Futura</family>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
    color: rgb(255, 255, 255);
	background-color: rgb(229, 88, 23);
    border-radius: 5px;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:hover{
    color: rgb(255, 255, 255);
	background-color: rgb(179, 70, 19);
    border-radius: 5px;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #31363b;
	background-color:#ffff00;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(225, 228, 228);
    background-color:  rgb(255, 255, 255);
    border-radius: 3;
    border: 2px solid rgb(225, 228, 228);
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>../WehaSmartPos/icons/keyboard.png</normaloff>../WehaSmartPos/icons/keyboard.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>16</width>
                  <height>16</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="spacing">
               <number>10</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLineEdit" name="PasswordEdit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>-1</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLineEdit{  
	background-color: rgb(204, 204, 204);
     border: 2px solid gray;
	border-radius: 10px;
	padding: 0 8px;
	selection-background-color: darkgray;
	font-size: 16px;
    color: rgb(255,255,255);
}
QLineEdit:focus {
     background-color: rgb(213, 83, 21);
     color: rgb(255,255,255);
}
</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="placeholderText">
                 <string>PASSWORD</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="PasswordNumpadButton">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Futura</family>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
    color: rgb(255, 255, 255);
	background-color: rgb(229, 88, 23);
    border-radius: 5px;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:hover{
    color: rgb(255, 255, 255);
	background-color: rgb(179, 70, 19);
    border-radius: 5px;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #31363b;
	background-color:#ffff00;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(225, 228, 228);
    background-color:  rgb(255, 255, 255);
    border-radius: 3;
    border: 2px solid rgb(225, 228, 228);
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>../WehaSmartPos/icons/keyboard.png</normaloff>../WehaSmartPos/icons/keyboard.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>16</width>
                  <height>16</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame{
	background-color: rgb(56, 67, 81);
	border-radius: 5px;
}
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="MessageLabel">
              <property name="font">
               <font>
                <family>Futura</family>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel {
	color: #000000
}</string>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="LoginButton">
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Futura</family>
                   <pointsize>14</pointsize>
                   <weight>50</weight>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
    color: rgb(255, 255, 255);
	background-color: rgb(229, 88, 23);
    border-radius: 5px;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:hover{
    color: rgb(255, 255, 255);
	background-color: rgb(179, 70, 19);
    border-radius: 5px;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #31363b;
	background-color:#ffff00;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(225, 228, 228);
    background-color:  rgb(255, 255, 255);
    border-radius: 3;
    border: 2px solid rgb(225, 228, 228);
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
                 </property>
                 <property name="text">
                  <string>LOGIN</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../WehaSmartPos/ui/icons/user.png</normaloff>../WehaSmartPos/ui/icons/user.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
